File: support_tools/swigwin-3.0.12/Examples/test-suite/python/preproc_runme.py
import preproc

if preproc.endif != 1:
    raise RuntimeError

if preproc.define != 1:
    raise RuntimeError

if preproc.defined != 1:
    raise RuntimeError

if 2 * preproc.one != preproc.two:
    raise RuntimeError

if preproc.methodX(99) != 199:
    raise RuntimeError

File: support_tools/swigwin-3.0.12/Examples/test-suite/javascript/preproc_runme.js
var preproc = require("preproc");

if (preproc.endif != 1)
  throw "RuntimeError";

if (preproc.define != 1)
  throw "RuntimeError";

if (preproc.defined != 1)
  throw "RuntimeError";

if (2*preproc.one != preproc.two)
  throw "RuntimeError";

